task_sdk/src/airflow/sdk/log.py

```diff
@@ -197,9 +193,19 @@ def logging_processors(
             exc_group_processor = None
 
         def json_dumps(msg, default):
+            # Note: this is likely an "expensive" step, but lets massage the dict order for nice
+            # viewing of the raw JSON logs.
+            # Maybe we don't need this once the UI renders the JSON instead of displaying the raw text
+            msg = {
+                "timestamp": msg.pop("timestamp"),
+                "level": msg.pop("level"),
+                "event": msg.pop("event"),
+                **msg,
+            }
```

Comment:
This is super useful, I have been meaning to get to it, as it was just harder to see the "event" normally. +1 for this change